src/App.vue

```diff
@@ -36,6 +36,7 @@ import VectorView from './components/VectorView';
 import config from './lib/config';
 import createDrag from './lib/utils/drag.js';
 import appState from './lib/appState';
+import { initAutoMode } from './lib/autoMode';
```

Comment:
Can we move this to the `nativeMain.js`? That way we wouldn't have to access a global `window.scene` variable - just pass it to the `autoMode` as a function argument

Reply:
Cool... Still learning my way around Vue!